scripts/aura_key.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
mmm having this versioned controlled might make us submit keys by mistake. perhaps add it as an example and the script not be depending on this specific file being there but just expect a json file as argument